pkg/git/repo_test.go

```diff
@@ -152,9 +152,25 @@ func TestCommitChanges(t *testing.T) {
 	require.Equal(t, 2, len(commits))
 	assert.Equal(t, "New commit with changes", commits[0].Message)
 
+	// Commit with trailers
+	trailers := map[string]string{
+		"test": "hoge",
+	}
+	changes2 := map[string][]byte{
+		"README.md": []byte("new-readme2"),
+	}
+	err = r.CommitChanges(ctx, "new-branch2", "New commit with changes and trailers", true, changes2, trailers)
+	require.NoError(t, err)
+
+	commits, err = r.ListCommits(ctx, "")
+	require.NoError(t, err)
+	require.Equal(t, 3, len(commits))
+	assert.Equal(t, "New commit with changes and trailers", commits[0].Message)
+
```

Comment:
[ASK] Is it possible to assert the trailers?

like: (just an example) 
```go
assert.Equal(t, "test: hoge", getTrailer(commits[0]))
```

Or you mean `NoError` of `CommitChanges()` is enough because it would fail if tailer is not supported?

Reply:
> Or you mean NoError of CommitChanges() is enough because it would fail if tailer is not supported?

I thought at that time, but it would be nice to check the trailers as you said 🙏 will fix it.

Reply:
@t-kikuc 
Trailers are set in the commit body. So I fixed to check the body itself.
[fb9344b](https://github.com/pipe-cd/pipecd/pull/5322/commits/fb9344b323d5d8dc930779cf06d2fd8101c99ce7)

Reply:
@ffjlabo Thank you!! It seems safer